### DISCUSSION TITLE
SiEPIC Ebeam SiN Layer Configuration Query

### DISCUSSION THREAD

**Discussion by Aydin1992** [SiEPIC/SiEPIC_EBeam_PDK#282] [General]
I'm using SiEPIC Ebeam technology for designing a SiN-based photonic circuit. I noticed that the SiN is defined on layer 1 and datatype 5 instead of layer 1 and datatype 0. Is there a specific reason for this? Could it be intended as an assistant (non-fabrication) layer?

![Untitled](https://github.com/SiEPIC/SiEPIC_EBeam_PDK/assets/43793279/6ea1ed37-3667-4ae7-a0f8-ae6498bdb9a5)

**Comment by lukasc-ubc** [comment]
Using separate layers for different fabrication steps is important so that there is no confusion.  So we chose 1/0 for Si and 1/5 for SiN.

**Reply by Aydin1992** [reply]
Thank you for your prompt reply.
The ANT design center can't recognize this layer, showing zero data because all polygons on layer 1 have a data type of 5. If I switch the layer to Silicon (1/0) but keep the shapes the same, would that fix the problem?

**Reply by mustafacc** [reply]
Yes I believe it should - ANT SiN NanoSOI uses the same layer for Si and SiN.

**Comment by lukasc-ubc** [comment]
You could also ask ANT to use a different layer number for different materials.

You can also remap the layers

**Comment by lukasc-ubc** [comment]
The PDK has been updated to use layer 4/0, which ANT has agreed to use for SiN fabrication.
